src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -73,8 +73,8 @@ public function __construct(CurlClientState $multi, $ch, array $options = null,
             $content = ($options['buffer'] ?? true) ? $content : null;
         }
 
-        curl_setopt($ch, CURLOPT_HEADERFUNCTION, static function ($ch, string $data) use (&$info, &$headers, $options, $multi, $id, &$location, $resolveRedirect, $logger): int {
-            return self::parseHeaderLine($ch, $data, $info, $headers, $options, $multi, $id, $location, $resolveRedirect, $logger);
+        curl_setopt($ch, CURLOPT_HEADERFUNCTION, static function ($ch, string $data) use (&$info, &$headers, &$options, $multi, $id, &$location, $resolveRedirect, $logger, &$content): int {
```

Comment:
does $option really need to be passed by reference? why? better not otherwise

Reply:
Removed the by reference passing